networkx/generators/geometric.py

```diff
@@ -72,8 +72,6 @@ def random_geometric_graph(n, radius, dim=2, pos=None, metric=None):
         If this argument is not specified, the Euclidean distance metric is
         used.
 
-    .. _metric: https://en.wikipedia.org/wiki/Metric_%28mathematics%29
```

Comment:
I didn't try indenting it... That works well and keep the URL in the section.
I made that change in commit d26d5e4
Thanks!